#! /bin/sh /usr/share/dpatch/dpatch-run
## pager.dpatch by Francesco Paolo Lovergine <frankie@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: As per debian wishes, choose the pager based on what 'pager' is
## DP: set to rather than the default of 'more'.
## DP: Problem with this patch: the default pager is usually 'less', which
## DP: is rather inappropriate for use with g.list where output is usally
## DP: less than 25 lines and it is useful to have the text remain on the
## DP: terminal after the pager is finished. IMO (HB) this is a diservice
## DP: to the user, who would be better served keeping the default as
## DP: 'more', or if that is unacceptable to debian, then use 'cat' and
## DP: have no pager turned on by default.
## DP: Note this in README.Debian??

@DPATCH@
diff -urNad grass-6.3~20070922~/lib/init/init.sh grass-6.3~20070922/lib/init/init.sh
--- grass-6.3~20070922~/lib/init/init.sh	2007-09-26 11:30:17.000000000 +0200
+++ grass-6.3~20070922/lib/init/init.sh	2007-09-26 11:32:48.000000000 +0200
@@ -229,10 +229,12 @@
 
 # Set some environment variables if they are not set
 if [ ! "$GRASS_PAGER" ] ; then
-    if [ -x /bin/more ] || [ -x /usr/bin/more ] ; then
-        GRASS_PAGER=more
+    if [ -x /usr/bin/pager ] ; then
+      	GRASS_PAGER=pager
     elif [ -x /bin/less ] || [ -x /usr/bin/less ] ; then
         GRASS_PAGER=less
+    elif [ -x /bin/more ] || [ -x /usr/bin/more ] ; then
+         GRASS_PAGER=more
     elif [ "$MINGW" ] ; then
         GRASS_PAGER=more
     else
